cmd/buildah/unshare.go

```diff
@@ -63,8 +63,7 @@ func maybeReexecUsingUserNamespace(c *cli.Context, evenForRoot bool) {
 	if c.NArg() == 0 {
 		return
 	}
-	switch c.Args()[0] {
-	case "help", "version":
+	if c.Args()[0] != "run" {
```

Comment:
is "run" the only command that requires a namespace?  I think any command that is accessing the storage requires multiple IDs to be available

Reply:
I don't think `buildah pull` will work without an user namespace